strtoint/strtoint2_test.go

```diff
@@ -0,0 +1,40 @@
+package strtoint
+
+import (
+	"github.com/stretchr/testify/assert"
+	"testing"
+)
+
+func Test_MyStrToInt2(t *testing.T) {
+	r, err := MyStrToInt2("150")
+
+	assert.Equal(t, r, 150, "should be equal")
+	assert.Nil(t, err, "should be nil")
```

Comment:
this package has a `NoError` func, which fail test if err is given. (like `assert.Error`, but mirrors it)